packages/controller/contracts/ANController.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity 0.5.17;
+
+import "./interfaces/IMiniMeLike.sol";
+import "./interfaces/ITokenController.sol";
+
+
+contract ANTController is ITokenController {
+    string private constant ERROR_NOT_MINTER = "ANC_SENDER_NOT_MINTER";
+
+    IMiniMeLike public ant;
+    address public minter;
+
+    event ChangedMinter(address indexed minter);
+
+    /**
+    * @dev Ensure the msg.sender is the minter
+    */
+    modifier onlyMinter {
+        require(msg.sender == minter, ERROR_NOT_MINTER);
+        _;
+    }
+
+    constructor(IMiniMeLike _ant, address _minter) public {
+        ant = _ant;
+        minter = _minter;
+    }
+
+    function mintANT(address _owner, uint256 _amount) external onlyMinter returns (bool) {
+        return ant.generateTokens(_owner, _amount);
+    }
+
+    function changeMinter(address _newMinter) external onlyMinter {
+        minter = _newMinter;
+        emit ChangedMinter(_newMinter);
```

Comment:
I'd extract this to an internal method and use it in the compiler as well

Reply:
Done in ac9aeba